docs/sphinx/conf.py

```diff
@@ -22,12 +22,12 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
```

Comment:
This templates sphinx config file has meaningful (not)spaces per line. Please revert.